projects/packages/forms/src/dashboard/class-dashboard.php

```diff
@@ -68,7 +68,7 @@ public function init() {
 	 * @param string $hook The current admin page.
 	 */
 	public function load_admin_scripts( $hook ) {
-		if ( 'toplevel_page_jetpack-forms' !== $hook ) {
+		if ( ! in_array( $hook, Dashboard_View_Switch::MODERN_SCREEN_IDS, true ) ) {
```

Comment:
should we use `is_modern_view()` here?

Reply:
`Dashboard_View_Switch` is not static, which means I should instance it and then call the method. I'm ok with that, just giving the rationale behind it. Are we ok with this? :

```php
		if ( ! ( new Dashboard_View_Switch() )->is_modern_view() ) {
			return;
		}
```

Reply:
Yep, that seems fine to me.